### ISSUE TITLE
pls help

### ISSUE THREAD

**Issue by chadius-bit** [Cryakl/Ultimate-RAT-Collection#19]
any rats with .bat output? orca gices an error

**Comment by Cryakl** [comment]
> any rats with .bat output? orca gices an error

No, not currently. You can make a .bat payload with an exe2bat crypter like Phantom.

https://github.com/C5Hackr/Phantom

**Comment by chadius-bit** [comment]
> > any rats with .bat output? orca gices an error
> 
> No, not currently. You can make a .bat payload with an exe2bat crypter like Phantom.
> 
> https://github.com/C5Hackr/Phantom

then can i smali it too?

**Comment by Cryakl** [comment]
> > > any rats with .bat output? orca gices an error
> > 
> > 
> > No, not currently. You can make a .bat payload with an exe2bat crypter like Phantom.
> > https://github.com/C5Hackr/Phantom
> 
> then can i smali it too?

Not sure what that is, but you can always test it for yourself.
